src/scripts/pagerduty.coffee

```diff
@@ -105,7 +105,7 @@ module.exports = (robot) ->
           if json.override
             start = moment(json.override.start)
             end = moment(json.override.end)
-            msg.send "Rejoice, #{old_username}! #{json.override.user.name} has the pager until #{end.format()}"
+            msg.send "Rejoice, @#{old_username}! @#{json.override.user.name} has the pager until #{end.format()}"
```

Comment:
I vaguely recall using the Slack API and having to do something special to get mentions working properly. I think you might need to know the user's ID if https://api.slack.com/docs/message-formatting#linking_to_channels_and_users is relevant here.